scripts/ci.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+case "$1" in
+  "build")
+    yarn lint
+    yarn build
+    ./scripts/check-schema.sh
+    ;;
+  "test")
+    yarn jest test/ --collectCoverage=true
+    yarn codecov
+    yarn jest examples/
+    yarn test:runtime
```

Comment:
Does run time depend on build? (Maybe not, just asking)

Reply:
No, I don't think so.